### PR TITLE
refactor: Destructure props into parameters and change class based co…

### PR DESCRIPTION
**Pull request by mhomolak**
…mponents to functional where possible

### CODE REVIEW

File: src/components/ProgressBar/ProgressBar.js [hunk 1]
```diff
@@ -1,54 +1,53 @@
...
 
-  calculateRatio() {
-    if (this.props.value < this.props.min) return this.props.min;
-    if (this.props.value > this.props.max) return this.props.max;
-    return Math.round((this.props.value - this.props.min) / (this.props.max - this.props.min) * 100);
-  }
+  return (
+    <div className={`progress-bar-container progress-bar-text-${alignLabel}`}>
+      <label className="pe-label" htmlFor={id}>
+        {calculateRatio()}{labelText}
```
Comment (StommePoes) [inline src/components/ProgressBar/ProgressBar.js]: might need a space between these (calc ratio and labelText), do they run together in the browser?

**Comment by StommePoes** [comment]
Looks good.

**Comment by StommePoes** [comment]
whoops :P

File: src/components/ProgressBar/ProgressBar.js (hunk 1 above)
Comment (mhomolak) [reply src/components/ProgressBar/ProgressBar.js]: I left them touching initially so something like "99% complete" could be done with the % complete being passed in by labelText.

Comment (StommePoes) [reply src/components/ProgressBar/ProgressBar.js]: ah ok that's cool

**Pull request merged**